lib/plugins/input/docker/docker.js

```diff
@@ -59,6 +59,15 @@ function InputDockerSocket (config, eventEmitter) {
       return (dockerInfo.LOGSENE_ENABLED === true)
     }
   }
+  // close log stream if any output filter emits the 'dropLogsRequest' event
+  var self = this
+  eventEmitter.on('dropLogsRequest', function (context, data) {
+    // consoleLogger.error('dropLogsRequest')
```

Comment:
Is this comment needed? If not, better to delete it.

Reply:
ok